src/test/groovy/org/mockito/release/notes/vcs/GitCommitSerializerTest.groovy

```diff
@@ -0,0 +1,31 @@
+package org.mockito.release.notes.vcs
+
+import org.apache.commons.lang.builder.EqualsBuilder
```

Comment:
I get it from other tests, and sorry but don't know how to check it. We got dependency `testCompile "commons-lang:commons-lang:2.4"` from which we got `EqualsBuilder`. Did you ask about that?